### DISCUSSION TITLE
Two MIDI Din IN & OUT ports

### DISCUSSION THREAD

**Discussion by gusarSlavko** [shanteacontrols/OpenDeck#307] [General]
Hi,
as the Raspberry Pico has two uart tx and rx (uart0 and uart1), would it be possible to have two Midi Din IN and OUT ports independent of each other? So for example one can just passthrough midi data from MIDI IN port 1 to MIDI OUT port 1, while OpenDerck sends all its midi data to MIDI OUT port 1. I hope this is written clearly :)
Thank you

**Comment by paradajz** [comment]
No.
